### ISSUE TITLE
Would you consider publishing to NuGet?

### ISSUE THREAD

**Issue by PeterKottas** [ollelogdahl/ConsoleGameEngine#21]
What a lovely project you have here! Really brings back memories. I was considering using this to teach my nieces some basic coding. Was wondering if you could publish to NuGet to make it easier for them to play with the lib (they are 10,9,7 y.o. so we're more in "click here" territory rather than referencing DLLs :) )

I am happy to provide guidance if need be. Cheers!

**Comment by WhiteBlackGoose** [comment]
This. By the way, you can reference them via git submodules and reference the csproj from your project, but nuget is still more convenient.

**Comment by PeterKottas** [comment]
Yeah, considering what it is, the ease of use is imho high priority. But I am afraid this is not maintained much anymore. Pitty.

**Comment by WhiteBlackGoose** [comment]
Yeah... @ollelogdahl, what do you think? If you need help with something, ask. Just don't leave such a great project! :)
